examples/arduino-mega2560/src/bin/mega2560-rgb-led.rs

```diff
@@ -0,0 +1,67 @@
+#![no_std]
```

Comment:
Please add a doc-comment explaining how the RGD LED should be wired to the board.

Reply:
done, please let me know if the doc comment is what you're looking for.

Reply:
Yeah, that's exactly what I wanted :)

I have one minor nit:  Please place the comment at the top of the file, using `//!` instead of `///`.  That's the way to make a doc-comment for the entire file.

Reply:
done, let me know if there's anything else. Looks like i rolled in some linting changes here, if thats problematic I can close this one in favor of a PR with just the new example.